pages/Useful Utilities/Systemd-start.md

```diff
@@ -0,0 +1,67 @@
+---
+weight: 12
+title: Systemd startup
+---
+
+## UWSM
+
+- [Universal Wayland Session Manager](https://github.com/Vladimir-csp/uwsm) is a recommended way to start Hyprland session on systemd distros. uwsm wraps standalone Wayland compositors into a set of Systemd units and provides robust session management including environment, XDG autostart support, bi-directional binding with login session, and clean shutdown.
+
+## Installation
+
+{{% details title="Arch" closed="true" %}}
+
+uwsm is available in AUR. You can install it either with aur helper, or build PKGBUILD manually. Here's an example for yay.
+
+```sh
+yay -S uwsm
+```
+
+{{% /details %}}
+
+{{% details title="Nix/NixOS" closed="true" %}}
+
+Available [here](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name/uw/uwsm/package.nix)
+
```

Comment:
```suggestion
\```nix
programs.uwsm = {
  enable = true;
  waylandCompositors.hyprland = {
    binPath = "/run/current-system/sw/bin/Hyprland";
    comment = "Hyprland session managed by uwsm";
    prettyName = "Hyprland";
  };
};
\```

The above option generates a new desktop entry, `hyprland-uwsm.desktop`, which will be available in Desktop Managers.

For more info, read the [option](https://search.nixos.org/options?channel=unstable&show=programs.uwsm.enable&from=0&size=50&sort=relevance&type=packages&query=uwsm).
```

Remove the backslashes, that was the only way to escpe the backticks.

I think I should create a dedicated page under Nix/ instead, as the setup is a bit different in some cases.